src/routes/protectedRoutes.tsx

```diff
@@ -0,0 +1,17 @@
+import BacktestForm from "../components/BacktestForm";
+import ContractScatterPlot from "../components/ContractScatterPlot";
+
+const protectedRoutes = [
```

Comment:
```suggestion
const protectedRoutes: Route[] = [
```

Reply:
moreover we can define all routes as an array, and keep the public protected information with each route. 
This will preserve the information and saves the code from getting diverged in future.
I.e., We might encounter someone who wants to handle the protected routes differently will just change the signature of those routes and slowly that will become divergent from the public routes.

To avoid that, we can keep a single list of routes and mark them public and protected via a boolean flag.